src/dataBase.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { Sequelize, DataTypes } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'test1234', {
+  host: 'localhost',
+  dialect: 'postgres',
+});
```

Comment:
i am glad that you trust us but would be better to hold creds in .env file and use variable here

Reply:
thank you for review!